googleapiclient/http.py

```diff
@@ -1252,11 +1252,10 @@ def _execute(self, http, order, requests):
     if resp.status >= 300:
       raise HttpError(resp, content, uri=self._batch_uri)
 
-    # Now break out the individual responses and store each one.
```

Comment:
I agree that this shouldn't be here, but it is not related to the rest of the commit/PR; move it to a separate commit/PR.